packages/tinacms/src/toolkit/react-sidebar/components/form-list.tsx

```diff
@@ -94,7 +95,7 @@ export const FormLists = (props: { isEditing: boolean }) => {
         enterFrom="opacity-0 -translate-x-1/2"
         enterTo="opacity-100"
         leave="transition-all ease-out duration-150"
-        className={'overflow-scroll'}
+        // className={"overflow-scroll"}
```

Comment:
**question**: Is this commented-out code worth retaining?

Reply:
Removed